convert-to-staging.php

```diff
@@ -0,0 +1,22 @@
+<?php
```

Comment:
This file do not need to be commit, it is part of docker. 
May be we can add in `.gitignore`

Reply:
Yeah. I already removed it.